assets/prototype/domain/eventEditor/tickets/ticketPriceCalculator/formDecorators/usePriceTypeDecorator.ts

```diff
@@ -1,33 +1,37 @@
+import { Calculation } from 'final-form-calculate';
 import { allPass, map, propEq, when } from 'ramda';
+
+import { TpcFormData } from '../types';
 import { isEqual, getFromFormData } from './utilities';
 import useTicketPriceCalculator from '../hooks/useTicketPriceCalculator';
+import { Price, PriceType } from '../../../data/types';
 import usePriceTypes from '../../../data/queries/priceTypes/usePriceTypes';
 import { getPriceType } from '../../../../shared/predicates/prices/selectionPredicates';
 import { getDefaultPriceType } from '../../../../shared/predicates/priceTypes/selectionPredicates';
 
-const usePriceTypeDecorator = () => {
-	const priceTypes = usePriceTypes();
+const usePriceTypeDecorator = (): Calculation => {
+	const priceTypes: PriceType[] = usePriceTypes();
```

Comment:
You don't need to add `: PriceType[]` there as it's automatically inferred by TS.

Reply:
ok will remove.